src/intents.cs

```diff
@@ -2294,6 +2806,13 @@ interface INInteraction : NSSecureCoding, NSCopying {
 
 		[NullAllowed, Export ("groupIdentifier")]
 		string GroupIdentifier { get; set; }
+
+		// From INParameter.h INInteraction ()
+
+		[iOS (11,0), Watch (4,0), NoMac]
+		[Export ("parameterValueForParameter:")]
+		[return: NullAllowed]
+		NSObject GetParameterValue (INParameter parameter);
```

Comment:
I wonder if this should be made `[Internal]` and expose an API with a generic type ?
This looks nicer than a cast but there's also the case where `(id)` can sometime be an `INativeObject` and not an `NSObject` subclass